services/121-service/seed-data/fsp/fsp-vodacash.json

```
@@ -23,6 +23,21 @@
         "en": "Phone Number",
         "fr": "Numéro de téléphone"
       }
+    },
+    {
+      "name": "healthArea",
+      "label": {
+        "en": "A.2 Health area",
+        "fr": "A.2 Aire de santé"
+      },
+      "answerType": "text",
+      "options": null,
+      "persistence": true,
+      "export": ["all-people-affected", "included"],
+      "shortLabel": {
+        "en": "A.2 Health area ",
+        "fr": "A.2 Aire de santé"
+      }
```

We don't _have_ to provide a `shortLabel`, right?
It will use the `label` by default, right?

```suggestion
      "export": ["all-people-affected", "included"]
```